astronomer/providers/snowflake/operators/snowflake.py

```diff
@@ -19,7 +19,34 @@
 
 class SnowflakeOperatorAsync(SnowflakeOperator):
     """
-    Executes SQL code in a Snowflake database
+    - SnowflakeAsyncOperator uses the snowflake python connector ``execute_async`` method to submit a database command
+      for asynchronous execution.
+    - Submit multiple queries in parallel without waiting for each query to complete.
+    - Accept List of queries or multiple queries with ‘;’ semicolon separated string and params. It loops through the
+      queries and execute the query in sequence. Uses execute_async method to run the query
+    - Once a query is submitted, it executes the query from one connection and gets the query id from the
+      response and passes it to the Triggerer and closes the connection (so that the worker slots can be freed up).
+    - The trigger gets the query id list as input and poll every few seconds to snowflake and check
+      for the query status based on the query id from different connection.
+
```

Comment:
Do we need `-` for the above statements? I guess we can remove those.

Reply:
Document doesn't look good Without `-`, Initially I added as paragraph it looks congested